cilium/tox.ini

```
@@ -4,7 +4,7 @@ skip_missing_interpreters = true
 basepython = py38
 envlist =
     py{27,38}-legacy
-    py{38}
+    py{38}-{1.9,1.10,1.11}
 
 [testenv]
 ensure_default_envdir = true
```

```suggestion
ensure_default_envdir = true
description =
    py{38}-{1.9,1.10,1.11}: e2e ready
```
To enable e2e tests again